tests/integration/kubernetes/tests_common.sh

```diff
@@ -80,20 +80,20 @@ auto_generate_policy_enabled() {
 	[ "${AUTO_GENERATE_POLICY}" == "yes" ]
 }
 
-# adapt common policy settings for tdx or snp
-adapt_common_policy_settings_for_tdx() {
+# adapt common policy settings for tdx, snp, or non-TEE dev environment
+adapt_common_policy_settings_coco() {
 	local settings_dir=$1
 
 	info "Adapting common policy settings for TDX, SNP, or the non-TEE development environment"
-	jq '.common.cpath = "/run/kata-containers" | .volumes.configMap.mount_point = "^$(cpath)/$(bundle-id)-[a-z0-9]{16}-"' "${settings_dir}/genpolicy-settings.json" > temp.json && sudo mv temp.json "${settings_dir}/genpolicy-settings.json"
+	jq '.common.cpath = "/run/kata-containers"' "${settings_dir}/genpolicy-settings.json" > temp.json && sudo mv temp.json "${settings_dir}/genpolicy-settings.json"
```

Comment:
Then we may need to `| .volumes.confidential_configMap.mount_point = "^$(cpath)/$(bundle-id)-[a-z0-9]{16}-"` in order to retain behaviour based on how confidential_guest is used in https://github.com/kata-containers/kata-containers/blob/main/src/tools/genpolicy/src/mount_and_storage.rs#L274 I may be on the wrong here

Reply:
Yeah, I'm currently trying to figure out
1. which platforms would currently need  `confidential_guest: true` and which don't. 
2. why those paths differ, i.e., in which configurations we need which set of paths/settings.
3. since `confidential_guest: true` currently doesn't align(?) with neither when an actual CVM is used (or emulated) or when `confidential_guest` is set in the kata config, I think we should rename the variable in genpolicy

Reply:
But to answer your question more directly:
> Then we may need to | .volumes.confidential_configMap.mount_point = "^$(cpath)/$(bundle-id)-[a-z0-9]{16}-" in order to retain behaviour based on how confidential_guest is used

As far as I understand, in this PR we have `confidential_guest: true` and therefore are using the `confidential_configMap` entry of the genpolicy settings, which use `sfprefix` and therefore `"^$(cpath)/$(bundle-id)-[a-z0-9]{16}-"`, i.e., the same path again. 
Though the `(confidential_)emptyDir` seems to currently cause problems.